lib/plaid.rb

```diff
@@ -51,12 +51,14 @@ def config
     #          as well.
     #
     # Returns a Hash with keys.to_sym (or nil if hash is nil).
-    def symbolize_hash(hash, values: false)
```

Comment:
Yes, it was: "Note that the calling code is syntactically equal to calling a method with hash arguments, which makes for an easy transition from options hashes to keyword arguments."
https://robots.thoughtbot.com/ruby-2-keyword-arguments

👍